### DISCUSSION TITLE
number of visitors

### DISCUSSION THREAD

**Discussion by FernandaMaria28** [Schweinepriester/github-profile-achievements#495] [Q&A] [answered]
friends, how can I put something that shows me how many people have visited my profile, I have seen in several profiles that have it if you could help me I would appreciate it very much.

**Comment by jaenfigueroa** [comment, marked as answer]
it's super easy, you just have to put this in your readme file.

![image](https://user-images.githubusercontent.com/69079292/202852919-50c8101c-8e67-4f6b-9e7b-4661b6e255a0.png)


what is in capital letters you change it to your user name

**Comment by jaenfigueroa** [comment]
![image](https://user-images.githubusercontent.com/69079292/202852756-ef87f5f6-dffa-41df-ace5-e781885122a2.png)
 
Here I send you an image

**Reply by FernandaMaria28** [reply]
Thank you :D
